cypress/support/commands.js

```diff
@@ -26,25 +26,40 @@
 
 Cypress.Commands.add('terminal', command => {
   const prefix = Cypress.env('COMMAND_PREFIX')
-  let cmd = `${command}`
-  if (prefix) {
-    cmd = `${prefix} ${cmd}`
-  }
-  cy.exec(cmd)
+  cy.exec(prefix ? `${prefix} ${command}` : command, {timeout: 60000})
 })
 
 Cypress.Commands.add('symfonyConsole', command => {
-  const prefix = Cypress.env('COMMAND_PREFIX')
-  let cmd = `bin/console ${command} --env="test"`
-  if (prefix) {
-    cmd = `${prefix} ${cmd}`
-  }
-  cy.exec(cmd)
+  cy.terminal(`bin/console ${command} --env="test"`)
+})
+
+Cypress.Commands.add('loadFixtures', (fixture, setup=false) => {
```

Comment:
The problem is that sometimes we want to load _multiple_ fixtures. This function should accept a variadic number of arguments.

Reply:
Did the change! It supports now a list of fixtures.
Applied over the added tests that were doing the manual call to `symfonyConsole`.